packages/cyberchef.vm/cyberchef.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>cyberchef.vm</id>
-    <version>10.18.3</version>
+    <version>10.18.3.20240523</version>
```

Comment:
It needs update because https://github.com/mandiant/VM-Packages/pull/1052 has been merged